components/Datasets/DatasetsMetrics.vue

```diff
@@ -97,46 +97,20 @@ const metricsReuses = ref<null | Record<string, number>>(null)
 const metricsReusesTotal = ref<null | number>(null)
 watchEffect(async () => {
   if (!props.organization.id) return
-
-  {
-    // Fetching last 12 months
-    const response = await fetch(`https://metric-api.data.gouv.fr/api/organizations/data/?organization_id__exact=${props.organization.id}&metric_month__sort=desc&page_size=12`)
-    const page = await response.json()
-
-    metricsViews.value = {}
-    metricsDownloads.value = {}
-    metricsReuses.value = {}
-
-    for (const { metric_month, monthly_download_resource, monthly_visit_dataset, monthly_visit_reuse } of page.data) {
-      metricsViews.value[metric_month] = monthly_visit_dataset
-      metricsDownloads.value[metric_month] = monthly_download_resource
-      metricsReuses.value[metric_month] = monthly_visit_reuse
-    }
-  }
-
-  {
-    // Fetching totals
-    const response = await fetch(`https://metric-api.data.gouv.fr/api/organizations_total/data/?organization_id__exact=${props.organization.id}`)
-    const page = await response.json()
-    if (page.data[0]) {
-      metricsViewsTotal.value = page.data[0].visit_dataset
-      metricsDownloadsTotal.value = page.data[0].download_resource
-      metricsReusesTotal.value = page.data[0].visit_reuse
-    }
-  }
+  const metrics = await getOrganizationMetrics(props.organization.id)
```

Comment:
I haven't followed this part but why do we get metrics with *organization* scope inside a DatasetMetrics components?

Reply:
This component was made to be on the datasets page of an organization in the administration but I agree that the name could be better.

But right now, we don't have other metrics to show because they aren't computed at the organization level.

The organization page should show different metrics when we have them.